src/components/ProjectsDropdown/index.tsx

```diff
@@ -4,12 +4,17 @@ import { Dropdown, DropdownItem } from '@terraware/web-components';
 
 import { useLocalization } from 'src/providers';
 import strings from 'src/strings';
-import { Project } from 'src/types/Project';
+
+type AvailableProject = {
+  name: string;
+  id: number;
+  [key: string]: any;
```

Comment:
Do we need this here? Seems like ProjectDropdown will not actually use them

Reply:
No, I think I was just struggling with typescript! Thanks